### ISSUE TITLE
Perform stage rollout for production track

### ISSUE THREAD

**Issue by ericliboon** [eventOneHQ/apkup#19]
Good day!

Would like to know if stage rollout can be supported using this?
https://developers.google.com/android-publisher/tracks

**Comment by nprail** [comment]
Yep, this is possible with `apkup`! You can use the `track` option to specify which track the APK should be uploaded to. 
```bash
npx apkup --apk ./app.apk --track alpha
```

Or you can promote existing APKs with the `promote` command. 

```bash
npx apkup promote --apk ./app.apk --track beta
```

**Comment by ericliboon** [comment]
Hi @nprail 
Thanks by the way for this awesome project! 
Sorry I haven’t clearly described from above the stage rollout i meant was for having the parameter to set a specific percentile for rollout in production track. 
For my use case, specially if there’s a major change developer applied on the version we had rolled out in alpha track, we perform production rollout to certain percentage only e.g. 20% then we gradually increase it till it reached full rollout if no issue was encountered.

**Comment by nprail** [comment]
Oh, I see! `apkup` does not currently support that. I'm going to guess adding it would be relatively straightforward, though!
